gateway/ln-gateway/src/lightning/mod.rs

```diff
@@ -108,6 +114,26 @@ pub trait ILnRpcClient: Debug + Send + Sync {
         &self,
         htlc: InterceptHtlcResponse,
     ) -> Result<EmptyResponse, LightningRpcError>;
+
+    async fn connect_to_peer(
```

Comment:
Add docstring to this function and the others

Reply:
Done.